### ISSUE TITLE
Search does not work when user selects a folder in the Advanced options

### ISSUE THREAD

**Issue by jelenadreskai** [MauroDataMapper/mdm-core#133]
Search does not work when user selects a folder in the Advanced options

1. Go to Mauro Main page
2. Click on Search in the top menu
3. Enter text  in the Search text field
4. Click on the Advanced button to select additional options
5. Select any folder where you know you have the Search term present

**Expected behavior**
Data catalogue items that match the search criteria in the selected folder would be shown in the search results

**Desktop (please complete the following information):**
 - OS: Windows OS
 - Browser: Chrome
 - Version:
> MDM_APPLICATION_COMMIT: 4.8.0
> MDM_UI_COMMIT: 6.4.0

**Screenshots**
> Showing search results with some items listed below.
![image](https://user-images.githubusercontent.com/81437684/129814146-5022a918-0757-4fbd-b7fd-a15afc2abc89.png)

> When I select the folder in which they reside, I get 0 in search results:
![image](https://user-images.githubusercontent.com/81437684/129814286-06fcd041-1de7-4210-ad6f-3979ca96d34a.png)

**Comment by pjmonks** [comment]
Just to double check, is your Lucene search index up to date when doing this? To rebuild the index from the UI, follow these instructions: https://maurodatamapper.github.io/installing/administration/#re-building-the-search-index

Does the issue still occur with a fully up to date search index?

**Comment by jelenadreskai** [comment]
Yes it does. I rebuild again to make sure, and I still get the same results.

**Comment by anthonyOCC** [comment]
Network requests show the same request and response whether the search restriction is present or not, so suspect this is a UI filtering issue, not backend.
